pkg/channel/distributed/receiver/README.md

```diff
@@ -4,12 +4,10 @@ The Receiver implementation is a Kafka Producer which is responsible for
 receiving CloudEvents, converting them to Kafka Messages, and writing them to
 the appropriate Kafka Topic.
 
-A unique Deployment / Service is created for every Kafka Secret (in the
-knative-eventing namespace and labelled with
-`eventing-kafka.knative.dev/kafka-secret: "true"`). The single Deployment is
-horizontally scalable as necessary (via controller environment variables.) This
-allows for an efficient use of cluster resources while still supporting high
-volume and multi-tenant use cases.
+A unique Deployment / Service is created for every installation.
+The single Deployment is horizontally scalable as necessary (via controller
+environment variables). This allows for an efficient use of cluster resources
+while still supporting high volume and multi-tenant use cases.
 
```

Comment:
maybe add something about users setting up their own HPA instead of the Controller setting replicas?

Reply:
Added a note to that effect